src/main/scala/net/snowflake/spark/snowflake/io/StageWriter.scala

```diff
@@ -319,6 +319,23 @@ private[io] object StageWriter {
     }
   }
 
+  private[snowflake] def getStageTableName(tableName: String): String = {
+    def genTableName():String =
+      s"spark_stage_table_${System.currentTimeMillis()}_${Math.abs(Random.nextInt)}"
+
```

Comment:
```scala
val pattern = "(.*\\.)?((\".+\")|([^\"]+))(\\s*)".r
tableName match {
    case pattern(dbAndSchema, _, _, _, _) => println(dbAndSchema)
    case _ => throw new Exception()
}
```

Reply:
try this. It can match `"aa".bb."cc""dd"`

Reply:
You are appreciated so much! It works.
I also add more unit test for table name including QUOTE character.